pkg/sdn/plugin/node.go

```diff
@@ -272,11 +272,13 @@ func (node *OsdnNode) Start() error {
 				continue
 			}
 			if vnid, err := node.policy.GetVNID(p.Namespace); err == nil {
-				node.policy.RefVNID(vnid)
+				node.policy.EnsureVNIDRules(vnid)
 			}
 		}
 	}
 
+	go kwait.Forever(node.syncVNIDRules, 24*time.Hour)
```

Comment:
Well, it's not clear to me how fast stale rules would build up, or how much of a problem they actually are (in terms of OVS performance). So it's hard to say whether once a day is too infrequent. But yeah, that was basically "random number". Anyone else want to vote?

I guess I could make it log how many stale entries it removes and then we can see what different clusters end up doing and adjust it in future releases.